pkg/ucglib/patches/0001-add-riot-makefiles.patch

```diff
@@ -0,0 +1,62 @@
+From 6bdd654ab1b904d245c57765262e1f6736224ddb Mon Sep 17 00:00:00 2001
```

Comment:
please just add these files to pkg/ucglib, and copy them in some Makefile recipe. No need to carry around patch files...

Reply:
Fixed